src/components/shared/StackedData.js

```diff
@@ -3,34 +3,48 @@ import PropTypes from "prop-types";
 import { Link } from "react-router-dom";
 import styled from "styled-components";
 
-const Wrapper = styled.td`
+const CellWrapper = styled.td`
   display: flex;
   flex-direction: column;
 `;
 
+const DivWrapper = styled.div`
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  padding: 12px;
+`;
+
 const Value = styled.span`
-  font-size: 12px;
+  font-size: ${props => props.cell ? "12px" : "16px"};
   word-wrap: break-word;
   word-break: break-all;
 `;
 
 const Label = styled.span`
   font-weight: 800;
-  font-size: 14px;
+  font-size: ${props => props.cell ? "14px" : "16px"};
 `;
 
-const StackedData = props => {
+const StackedBody = props => {
   return (
-    <Wrapper className="stacked">
-      <Label>{props.label}</Label>
-      <Value>
+    <>
+      <Label {...props}>{props.label}</Label>
+      <Value {...props}>
         {props.link ? <Link to={props.link}>{props.value}</Link> : props.value}
```

Comment:
I didn't think of this until I was finished refactoring, but we might be able to get rid of the `value` prop, and instead just use `children` here. Might be worth a second refactor?

Reply:
+1 Yes, I think that would be a nice DX improvement - only keeping Label as a prop.